test/enhance/test_jpeg.py

```diff
@@ -0,0 +1,990 @@
+                            -0.000,
+                            0.020,
+                        ],
+                        [
+                            0.060,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            0.173,
+                            0.246,
+                            0.178,
+                            0.080,
+                            0.080,
+                            0.178,
+                            0.246,
+                            0.173,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            0.060,
+                        ],
+                        [
+                            0.063,
+                            0.009,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            0.009,
+                            0.063,
+                        ],
+                        [
+                            0.002,
+                            0.015,
+                            0.008,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            -0.000,
+                            0.008,
+                            0.015,
+                            0.002,
+                        ],
+                        [
+                            -0.000,
+                            0.002,
+                            0.063,
+                            0.060,
+                            0.020,
+                            0.017,
+                            0.078,
+                            0.146,
+                            0.146,
+                            0.078,
+                            0.017,
+                            0.020,
+                            0.060,
+                            0.063,
+                            0.002,
+                            -0.000,
+                        ],
+                    ],
+                ]
+            ],
+            device=device,
+            dtype=dtype,
+        )
+        # We use a slightly higher tolerance since our implementation varies from the reference implementation
+        assert_close(img_jpeg[..., :4, :4], img_jpeg_ref[..., :4, :4], rtol=0.01, atol=0.01)
```

Comment:
```suggestion
        self.assert_close(img_jpeg[..., :4, :4], img_jpeg_ref[..., :4, :4], rtol=0.01, atol=0.01)
```
coming from `BaseTester`